paddle/fluid/operators/matmul_op.cc

```diff
@@ -448,6 +486,14 @@ class MatMulOpMaker : public framework::OpProtoAndCheckerMaker {
         "use_mkldnn",
         "(bool, default false) Indicates if MKL-DNN kernel will be used")
         .SetDefault(false);
+    AddAttr<std::vector<int>>(
+        "fused_reshape_Out",
+        R"DOC(Shape of fused reshape of `Out` output.)DOC")
+        .SetDefault({});
+    AddAttr<std::vector<int>>(
+        "fused_transpose_Out",
+        R"DOC(Axis of fused transpose of `Out` output.)DOC")
+        .SetDefault({});
```

Comment:
Is this necessary to add two attributes in matmul_op? And the comments are not enough, maybe you can improve the comments in the next PR?

Reply:
It would be nice to see a pair of vectors for example because we require both attributes present anyway. Hower, we didn't want to extend the Attribute definition or hack it. 
https://github.com/PaddlePaddle/Paddle/blob/a31d7328b70fde9c860dfad918222d7b2d842d71/paddle/fluid/framework/type_defs.h#L41-L45

Reply:
I can improve comments in the next PR